mailer.js

```diff
@@ -280,17 +283,26 @@ module.exports = function (log) {
       headers['X-Flow-Begin-Time'] = message.flowBeginTime
     }
 
+    if (message.service === 'sync') {
+      subject = 'Confirm your email and start to sync!'
+      templateName = 'verifySyncEmail'
+    }
+
     return this.send({
       acceptLanguage: message.acceptLanguage,
       email: message.email,
       headers: headers,
-      subject: gettext('Verify your Firefox Account'),
+      subject: gettext(subject),
```

Comment:
@vbudhram this `gettext(subject)` will not work for string extraction, the strings need to be written out as `gettext('Stuff here')` in the code